cloud/deployment/workerqueue/workerqueue.go

```diff
@@ -49,15 +51,31 @@ func CreateOrUpdate(ws, deploymentID, deploymentName, name, action, workerType s
 		return err
 	}
 
-	// get the node poolID to use
-	nodePoolID, err = selectNodePool(workerType, requestedDeployment.Cluster.NodePools, out)
-	if err != nil {
-		return err
+	if deployment.IsDeploymentHosted(requestedDeployment.Type) {
+		nodePoolID = requestedDeployment.Cluster.NodePools[0].ID
```

Comment:
just trying to understand the logic, we picked the first element directly because there is just a default node pool on Hosted as of now?